pushapkscript/googleplay.py

```diff
@@ -3,11 +3,11 @@
 from scriptworker.exceptions import TaskVerificationError
 from scriptworker.utils import get_single_item_from_sequence
 
-from pushapkscript.task import extract_channel
+from pushapkscript.task import extract_android_product_from_scopes
 
 log = logging.getLogger(__name__)
 
-_CHANNELS_AUTHORIZED_TO_REACH_GOOGLE_PLAY = ('aurora', 'beta', 'release')
+_AUTHORIZED_PRODUCTS_TO_REACH_GOOGLE_PLAY = ('aurora', 'beta', 'release', 'focus')
```

Comment:
Do we need a `dep` here as well?

Reply:
Nope, `dep` should avoid contacting Google Play. We haven't registered any app signed by the dep key there. Therefore, the task will report red.